### PR TITLE
delete extra file

### PR DESCRIPTION
**Pull request by Tahoora78**
remove

### CODE REVIEW

File: attendees [hunk 1]
```diff
@@ -1,4 +0,0 @@
-
-faeze mahmoudi
```
Comment (ketabi) [inline attendees:2]: Why you deleted this line?

**Review by ketabi — CHANGES_REQUESTED** [review]

File: attendees (hunk 1 above)
Comment (Tahoora78) [reply attendees:2]: Cause attendees was duplicated

**Comment by ketabi** [comment]
```curl ifconfig.io```

**Comment by ketabi** [comment]
two books for python :
- distilled david beazley
- fluent ramlho

and two books for js:
- you dont know js [Kyle Simpson]
- The difinnitive guide js david flanagan

**Pull request closed without merging**